man/systemd.network.xml

```diff
@@ -1753,6 +1753,35 @@
         </varlistentry>
       </variablelist>
   </refsect1>
+
+  <refsect1>
+    <title>[CAN] Section Options</title>
+      <para>The <literal>[CAN]</literal> section manages the Controller Area Network (CAN bus) accepts
+      the following keys.</para>
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>BitRate=</varname></term>
+          <listitem>
+            <para>The bitrate of CAN device in bit per second.</para>
```

Comment:
might be worth saying that the usual  K, M, G, … suffixes are accepted, to the base of 1000

Reply:
ok